vowel-count-task.js

```diff
@@ -0,0 +1,10 @@
+// Napisz funkcję vowelCount(), która zwróci liczbę samogłosek w podanym łańcuchu
+// znaków.
+
+function vowelCount(string) {
+    const vowels = string.match(/[aeiouy]/gi);
```

Comment:
That's a good name since it's an array of found vowels! :)

Another good name would be for example `matches`.